pkg/controllers/management/auth/user.go

```diff
@@ -308,3 +321,21 @@ func (l *userLifecycle) deleteAllTokens(tokens []*v3.Token) error {
 
 	return nil
 }
+
+func (l *userLifecycle) deleteUserNamespace(username string) error {
+	namespace, err := l.namespaceLister.Get("", username)
+	if err != nil {
```

Comment:
One of the errors can be an not exists or not found error (i forget which). if we hit that error, can you _not_ bubble it up?